packages/shared/src/utils/constants/contract.tsx

```diff
@@ -429,11 +438,13 @@ export const getSupportedContracts = (is_high_low?: boolean) => ({
         position: 'bottom',
     },
     TURBOSLONG: {
-        name: <Localize i18n_default_text='Buy' />,
+        name: <Localize i18n_default_text='Tubos' />,
```

Comment:
typo

Reply:
The types were described above: `TContractConfig`(27 line) and `TGetSupportedContracts`(33 line). We use them inside function:
```
export const getContractTypeDisplay = (type: TGetSupportedContracts, is_high_low = false, show_trade_text = false) => {
    return show_trade_text
        ? (getContractConfig(is_high_low)[type] as TContractConfig).button_name
        : getContractConfig(is_high_low)[type].name;
};
```

Reply:
No I mean there is a typo in `Tubos` word, it should be `Turbos` 😋

Reply:
Ooooo, I see, sorry about that)